pkg/ottl/internal/ottlcommon/value.go

```diff
@@ -37,56 +37,3 @@ func GetValue(val pcommon.Value) interface{} {
 	}
 	return nil
 }
-
-func SetValue(value pcommon.Value, val interface{}) {
```

Comment:
SetValue has a dependency on SetMapValue, so I can move them all back into `ottlcommon`.

Reply:
Oh right, SetEmptyMap has a dependency on `ottl.keys` at the moment and I don't want to move anything from the grammar into ottlcommon and I don't really want to make a separate struct for this yet since I think that will arise once we use allow non-string accessing.  Lets keep this "optimal" at the moment.  Since GetValue is the only shared thing its the only part that needs to be in ottlcommon for now.

Reply:
Okay, that makes sense. Sounds good to me.